ImageInLib/ImageInLib/src/fast_sweeping_method.c

```diff
@@ -39,7 +42,9 @@ bool fastSweepingFunction_3D(dataType ** distance3DPtr, dataType ** curve3DPtr,
 	//Checks for 2D case
 	if (zDim == 1)
 		return false;
-
+	//==============================================================================
+	// curve3DPtr is in the range 0-1 binary
```

Comment:
I do not understand the comment.
What do you mean by "0-1 binary". Is itbinary or in range <0,1>?

Reply:
Range 0 to 1